linux_various/PCSX2.desktop.in

```diff
@@ -5,7 +5,7 @@ Type=Application
 Name=PCSX2
 GenericName=Playstation 2 Emulator
 Comment=Sony Playstation 2 emulator
-Exec=pcsx2
-Icon=pcsx2
+Exec=PCSX2
```

Comment:
I want people to execute PCSX2-linux.sh, this way nvidia variable is correctly set (if you have a better solution I take it)